pkg/scheduler/framework/plugins/noderesources/fit.go

```diff
@@ -97,6 +101,39 @@ func (s *preFilterState) Clone() framework.StateData {
 	return s
 }
 
+// preScoreState computed at PreScore and used at Score.
+type preScoreState struct {
+	podRequest map[v1.ResourceName]int64
+}
+
+// Clone implements the mandatory Clone interface. We don't really copy the data since
+// there is no need for that.
+func (s *preScoreState) Clone() framework.StateData {
+	return s
+}
+
+// PreScore calculates incoming pod's resource requests and writes them to the cycle state used.
+func (f *Fit) PreScore(ctx context.Context, cycleState *framework.CycleState, pod *v1.Pod, nodes []*v1.Node) *framework.Status {
+	state := &preScoreState{
+		podRequest: f.calculatePodResourceRequestMap(pod, f.resources),
+	}
+	cycleState.Write(preScoreStateKey, state)
+	return nil
```

Comment:
`Fit` plugin as well. Please check the [all scorer](https://github.com/kubernetes/kubernetes/blob/268c02a1c7f63d760af7c2bf614e93f95ea80e9d/pkg/scheduler/framework/plugins/noderesources/fit.go#L54-L79) to see if we can return skip in some cases.

Reply:
I think `Fit` plugin should return skip if Pod has no resource request too.